temporal-testing/README.md

```diff
@@ -44,4 +44,28 @@ and `io.temporal.testing.TestActivityExtension` for isolated testing of activiti
 
 See `io.temporal.testing.TestActivityEnvironment` that provides an easy way for isolated testing of
 activity implementations without needing to provide workflows calling the activities, triggering the workflows
-and bootstrapping a Temporal server or In-memory testing service.
\ No newline at end of file
+and bootstrapping a Temporal server or In-memory testing service.
+
+## To build a test service using GraalVM native-image
+
+From the root of the java-sdk repo:
+```
+./gradlew :temporal-testing:copyDependencies
+```
+```
+native-image -jar "./temporal-testing/build/libs/temporal-testing-1.6.0-SNAPSHOT.jar" test-server --class-path "./temporal-testing/build/dependencies/*" \
+--allow-incomplete-classpath --no-fallback \
+--initialize-at-build-time=org.slf4j.impl.StaticLoggerBinder \
+--initialize-at-build-time=org.slf4j.LoggerFactory \
+--initialize-at-run-time=io.grpc.netty.shaded.io.netty.util.internal.logging.Log4JLogger \
+--initialize-at-run-time=io.grpc.netty.shaded.io.netty.internal.tcnative \
+--initialize-at-run-time=io.grpc.netty.shaded.io.netty.handler.ssl
```

Comment:
Pardon my lack of knowledge of native-image. Can you help me understand the use of `native-image.properties` and manual CLI arguments? What purpose does the properties file serve if these arguments are also here?

Reply:
Well, users of your library pick them up. So, native-image will inspect the underlying jars for manifests.
I think there is a way to don't have them there and make the properties file to be used. But I wasn't able to make it work just yet.